### ISSUE TITLE
Tag is null

### ISSUE THREAD

**Issue by andidev** [notifee/react-native-notifee#382]
When displaying a notification with tag
```js
  notifee.displayNotification({
      title: 'Title',
      body: 'Body',
      android: {
          channelId: 'DELIVERY',
          tag: 'DELIVERY',
      },
  });
```

and the calling 
```js
await notifee.getDisplayedNotifications();
```

notification tag is null when it should be `DELIVERY`
```json
[
    {
        "notification": {
            "title": "Title",
            "id": "165401543",
            "body": "Body",
            "android": {
                "channelId": "DELIVERY",
                "group": null,
                "tag": null
            }
        },
        "date": "1631268354598",
        "id": "165401543"
    }
]
```

**Comment by andidev** [comment]
Also creating multiple local notifications with tags does not only keep last notification as stated in docs. Maybe tags is not supposed to work for local notifications?

**Comment by helenaford** [comment]
Ah, thanks for reporting the issue. sorry for the inconvenience. I've just taken a look. I can see why this would be the case. Will mark this as a bug.

 We are also working on a way to cancel by tag which will be needed if we fix this.

**Comment by helenaford** [comment]
fixed in release v3.0.0 (https://notifee.app/react-native/docs/release-notes#300)
